adam/curriculum_to_html.py

```diff
@@ -0,0 +1,184 @@
+import os
+from typing import List
+
+from attr import attrs
+
+from adam.experiment import InstanceGroup
+from adam.language.dependency import LinearizedDependencyTree
+from adam.perception import PerceptualRepresentation
+from adam.perception.developmental_primitive_perception import (
+    DevelopmentalPrimitivePerceptionFrame,
+)
+from adam.situation.high_level_semantics_situation import HighLevelSemanticsSituation
+
+
+@attrs(frozen=True, slots=True)
+class CurriculumToHtml:
+    """
+    Class to turn an `InstanceGroup` into an html document
+    """
+
+    @staticmethod
+    def generate(
+        instances: List[
```

Comment:
Prefer `Sequence` over `List` as an argument type - that way the use has more flexibility. Even better here is just plain `Iterable`.

Reply:
Also it looks like this should be called `instance_groups` instead of `instances`?